src/main/kotlin/view/ResultView.kt

```diff
@@ -0,0 +1,28 @@
+package view
+
+import model.Race
+
+class ResultView {
+    private val mark = "-"
+
+    fun showResult(record: List<Race.Record>) {
+        display(record)
+        println()
+    }
+
+    private fun display(result: List<Race.Record>) {
+        result.forEach {
+            println("${it.name}: ${mark.repeat(it.distance)}")
+        }
+    }
+
+    fun whoIsWinner(result: List<Race.Record>) {
+        val maxDistance = result.maxOf { it.distance }
+        val winnersName = result.filter { it.distance == maxDistance }
+            .map { it.name }
+
+        println("${winnersName.list()}가 최종 우승했습니다.")
+    }
+
+    private fun List<String>.list() = this.reduce { first, next -> "$first, $next" }
```

Comment:
확장함수 활용 👍👍
```suggestion
    private fun List<String>.list() = joinToString(",") { it }
```
이런 방법은 어떠신가요?

참고로 이펙티브 코틀린에서는 확장함수를 클래스 스코프 밖에 선언하기를 권장하고 있습니다. 
https://kt.academy/article/ek-member-extensions